Integrations/integration-Kafka_V2.yml

```diff
@@ -95,14 +96,19 @@ script:
     BROKERS = demisto.params().get('brokers')
 
     # Should we use SSL
-    USE_SSL = demisto.params().get('insecure', False)
+    USE_SSL = not demisto.params().get('insecure')
```

Comment:
need to have default: `VERIFY_SSL = not demisto.params().get('insecure', False)`